src/test/CMakeLists.txt

```diff
@@ -414,6 +412,7 @@ target_link_libraries(
   repo_agent_test
   PRIVATE
     triton-common-error       # from repo-common
+    triton-common-model-config# from repo-common
```

Comment:
Add a space before '#'

Reply:
@jbkyang-nvi  other that this one small change it looks ok to me. But you should review and run CI...

Reply:
Done